### PR TITLE
SCCP-322-323-324-325

### PR DESCRIPTION
**Pull request by kaleb-keny**

### CODE REVIEW

File: tomls/omnibus-base-mainnet-andromeda/perps/eth.toml [hunk 1]
```diff
@@ -20,7 +20,7 @@ defaultValue = "0.0005"
 defaultValue = "2777" # approx $10m
 
 [setting.perpsEthMaxMarketValue]
-defaultValue = "0" # not enabled
+defaultValue = "7000000" # not enabled
```
Comment (noisekit) [inline tomls/omnibus-base-mainnet-andromeda/perps/eth.toml:23]: Should comment change too?

File: tomls/omnibus-base-mainnet-andromeda/perps/ena.toml [hunk 2]
```diff
@@ -5,7 +5,7 @@ defaultValue = "0xb7910ba7322db020416fcac28b48c01212fd9cc8fbcbaf7d30477ed8605f6b
 defaultValue = "700"
 
 [setting.perpsEnaSkewScale]
-defaultValue = "34000000"
+defaultValue = "25500000"
```
Comment (noisekit) [inline tomls/omnibus-base-mainnet-andromeda/perps/ena.toml:8]: Here and for all other similar places: for bigger numbers for readability it is worth using different format
```suggestion
defaultValue = "<%= String(25_500_000) %>"
```

Comment (kaleb-keny) [reply tomls/omnibus-base-mainnet-andromeda/perps/ena.toml:8]: this is pretty nice tbh, yes

File: tomls/omnibus-base-mainnet-andromeda/perps/eth.toml (hunk 1 above)
Comment (kaleb-keny) [reply tomls/omnibus-base-mainnet-andromeda/perps/eth.toml:23]: yes sure will update

Comment (kaleb-keny) [reply tomls/omnibus-base-mainnet-andromeda/perps/eth.toml]: done

File: tomls/omnibus-base-mainnet-andromeda/perps/ena.toml (hunk 2 above)
Comment (kaleb-keny) [reply tomls/omnibus-base-mainnet-andromeda/perps/ena.toml:8]: will do it on all after we merge this. for the sake of no bad inputs.

**Review by noisekit — APPROVED** [review]

**Pull request merged**